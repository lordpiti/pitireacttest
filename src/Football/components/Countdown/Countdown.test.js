import React from 'react';
import renderer from 'react-test-renderer';
import Countdown from './Countdown';

describe('<MatchPlayerSingle /> with all data', () => {
	it('renders correctly a player component with the right data', () => {

		const date = '2019-04-04';
		const tree = renderer.create(<Countdown date={date} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
})