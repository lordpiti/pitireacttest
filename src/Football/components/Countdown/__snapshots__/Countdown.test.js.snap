// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MatchPlayerSingle /> with all data renders correctly a player component with the right data 1`] = `
<div
  className="Countdown"
>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Days
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Hours
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Min
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Sec
      </span>
    </span>
  </span>
</div>
`;
